backend/src/controllers/notificationController.js

```diff
@@ -0,0 +1,76 @@
+const Notification = require('../models/notificationModel')
+
+//Handle get all notification
+exports.getNotifications = function(req, res) {
+    Notification.find({ user: req.params.user }, function(err, notifications) {
+        if (err) {
+            res.status(500).send(err);
+        }
+        res.status(200).json({
+            data: notifications
+        });
+    });
+}
+
+//Handle view get notification by id
+exports.getNotificationById = function(req, res) {
+    Notification.findById(req.params.notification_id, function(err, notification) {
+        if (err) {
+            res.status(500).send(err);
+        }
+        res.status(200).json({
+            data: notification
+        });
+    });
+}
+
+//Handle create new notification
+exports.createNotification = function(req, res) {
+    var nt = new notification();
```

Comment:
wait, how does notifications with lower case n work? Isn't javascript case sensitive?

Reply:
Sorry, I wanted to write **Notification**